src/test/java/spoon/test/annotation/AnnotationTest.java

```diff
@@ -1547,4 +1547,17 @@ public void testAnnotationArray() throws Exception {
 		assertTrue(shadowAnnotationOne.getValue("role") instanceof CtLiteral); // should be CtLiteral
 		assertEquals(annotationOne.getValue("role"), shadowAnnotationOne.getValue("role")); // should pass
 	}
+
+	@Test(expected = SpoonException.class)
+	public void testGetValueAsObject() {
+		// contract: when one cannt evaluate an expression, throw an exception
```

Comment:
cannt -> cannot

Reply:
Thanks, but already merged. Could you do a live pull-request with web-based editing?